app/views/shared/_banner.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="banner"">
+    <%= render "shared/navbar" %>
+    <div class="text">
+        <h1>Your Body, Your Practice</h1>
+        <p>DISCOVER YOURSELF, DISCOVER YOGA</p>
+        <button class="btn btn-gradient">Start now</button>
+    </div>
+</div>
```

Comment:
The indentation should be 2 spaces, not 4

also please add an empty space between the navbar and the div.text to space this html a bit. It'll make it easier to read.
You also need to wrap the navbar and the div.text inside a div.container

Also this partial will only every be used inside the home page so please move it inside the views/pages/partials folder